AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -260,8 +288,24 @@ struct ChooseSessionTypeView: View {
                 .foregroundColor(.aircastingGray)
                 .multilineTextAlignment(.leading)
         }
-        .padding()
-        .frame(maxWidth: 180, maxHeight: 145)
+        .padding(15)
+        .frame(maxWidth: 147, maxHeight: 145)
+        .background(Color.white)
+        .shadow(color: Color.shadow, radius: 9, x: 0, y: 1)
+    }
+    
+    var syncButtonLabel: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            Text(Strings.ChooseSessionTypeView.syncTitle)
+                .font(Fonts.boldHeading1)
+                .foregroundColor(.accentColor)
+            Text(Strings.ChooseSessionTypeView.syncDescription)
+                .font(Fonts.muliHeading3)
+                .foregroundColor(.aircastingGray)
+        }
+        .multilineTextAlignment(.leading)
+        .padding(15)
+        .frame(maxWidth: 147, maxHeight: 145)
```

Comment:
For some reason this button is wider than the rest fo the buttons

Reply:
<img width="359" alt="Screenshot 2021-12-03 at 16 59 48" src="https://user-images.githubusercontent.com/44276070/144642398-0af2f9e0-e93d-4ddf-a6aa-7cd0061ca917.png">

Reply:
It's the same I think and I checked